lib/fedex_rest_api/track.rb

```diff
@@ -0,0 +1,40 @@
+require "fedex_rest_api/base"
+require "debug"
+
+class FedexRestApi::Track
+  attr_reader :tracking_object
+
+  def initialize(tracking_object)
+    @tracking_object = tracking_object
+  end
+
+  def track
+    response = HTTParty.post("#{env_url}",
```

Comment:
Does this work without interpolation, `.post(env_url,...`?

Reply:
Yup! Good catch.